raster-foundry/raster-foundry
# Make sure actions get updated in label post

## Overview

This PR removes the ad hoc method for updating a task status and runs everything through `updateTask`. This ensures that actions are always updated.

### Checklist

- [x] Any new SQL strings have tests (✅ in the sense of _removing_ tests for things that no longer exist 😎)

### Notes

Unclear what to do to verify that this fixes the current problem -- I'd like help with that if someone has a minute

## Testing Instructions

- confirm some labels for a task with GW pointed to local RF
- check the actions for that task
- you should see an action for when it was set to labeled
